gnomad_qc/v3/load_data/compute_coverage.py

```diff
@@ -33,13 +36,13 @@ def main(args):
         coverage_ht = coverage_ht.checkpoint('gs://gnomad-tmp/gnomad.genomes_v3.coverage.summary.ht', overwrite=True)
         coverage_ht = coverage_ht.naive_coalesce(5000)
 
-        coverage_ht.write(coverage('genomes').versions['3.0'].path, overwrite=args.overwrite)
+        coverage_ht.write(coverage('genomes').versions['3.0.1'].path, overwrite=args.overwrite)
```

Comment:
instead of hardcoding the version, maybe you could specify a default value (`CURRENT_GENOME_COVERAGE_RELEASE`)?